Makefile

```diff
@@ -219,7 +211,7 @@ PSLITE:
 $(DMLC_CORE)/libdmlc.a: DMLCCORE
 
 DMLCCORE:
-	+ cd $(DMLC_CORE); make libdmlc.a USE_SSE=$(USE_SSE) config=$(ROOTDIR)/$(config); cd $(ROOTDIR)
```

Comment:
was a change mistakenly introduced by trying to use copy paste instead of merging, will resolve this